sklearn/datasets/_kddcup99.py

```diff
@@ -88,7 +90,7 @@ def fetch_kddcup99(*, subset=None, data_home=None, shuffle=False,
         Whether to load only 10 percent of the data.
 
     download_if_missing : bool, default=True
-        If False, raise a IOError if the data is not locally available
+        If False, raise a OSError if the data is not locally available
```

Comment:
The argument should be documented here in the parameters section.

Reply:
Im sorry, what do you mean? which argument?

Reply:
the ``as_frame`` argument.

Reply:
OK, will add :)

Reply:
thanks, I think it would be good if you also modify the "returns" section as you did in the other PR, sorry I didn't mention that earlier.

Reply:
Yep DONE.